src/typings/react-native.d.ts

```diff
@@ -397,6 +409,8 @@ declare module 'react-native' {
         textBreakStrategy?: 'highQuality' | 'simple' | 'balanced';
         // macOS only property for submitting the text on enter
         submitTextOnEnter?: boolean;
+        // Windows only
+        tabIndex?: number;
```

Comment:
Is tabIndex really needed for ScrollView? If so, why is it not applicable to web as well?

Reply:
This one is part of the RN.TextInputProps, and yes, it's the Windows flavor of RN that supports it.
RX.TextInput maps to <textarea> or <input> in the web implementation. RX.TextInputProps does contain tabIndex, which is not conveyed to the HTML elements (it should!), yet the web flavor of FocusManager does set the tabIndex DOM property on those elements liberally.
We needed something similar in RN for W. The resultant of RX.TextInputProps.tabIndex & FocusManager overrides is pushed to RN.TextInput through this property.

---
In reply to: [159786215](https://github.com/Microsoft/reactxp/pull/404#discussion_r159786215) [](ancestors = 159786215)